x-pack/plugin/apm-data/src/main/resources/ingest-pipelines/apm@pipeline.yaml

```diff
@@ -24,3 +24,13 @@ processors:
     target_field: user_agent
     ignore_failure: true
     ignore_missing: true
+- script:
```

Comment:
@inge4pres This pipelines is used for all data streams, metrics included, but you added tests only for logs and traces. Should an additional test for metrics data streams be added?

Reply:
Ok I can do that 👍🏼

Reply:
Added in last commit